api/src/main/java/jakarta/data/repository/CrudRepository.java

```diff
@@ -83,6 +84,68 @@ public interface CrudRepository<T, K> extends DataRepository<T, K> {
      */
     <S extends T> Iterable<S> saveAll(Iterable<S> entities);
 
+    /**
+     * <p>Inserts an entity into the database. If an entity of this type with the same
+     * unique identifier already exists in the database, then this method raises
+     * {@link EntityExistsException}.</p>
+     *
+     * @param entity the entity to insert. Must not be {@code null}.
+     * @throws EntityExistsException if the entity is already present in the database.
+     * @throws NullPointerException if the entity is null.
+     */
+    void insert(T entity);
```

Comment:
IMHO: those methods should only be available dynamically, as discussed in the meeting.

By default, just the save makes sense.

Reply:
I'm pretty sure we said the opposite during the meeting, that CRUD in CrudRepository stands for Create, Read, Update, and Delete.  The `insert` method is the Create.  The `update` method is the Update.  And it appears very odd for those to be missing from something that calls itself CRUD.

Let me know if you insist on separating these out into a separate pull.